src/redux/reducers/beersReducer.js

```diff
@@ -28,6 +34,25 @@ export default function beersList(state = initialState, action) {
         error: action.payload.error,
       };
     }
+    case FILTER_BEERS: {
+      const { beersList: beers, searchedBeers } = state;
+      const { ibu, abv, ebc } = action.payload;
+
+      const searched = [...(searchedBeers ?? beers)];
+
+      const filtered = searched.filter(
```

Comment:
to filter beers by values you can use query params to fetch only beers with required values

Please have a look at [Official Punk API docs](https://punkapi.com/documentation/v2)